unittests/LoggerTests.cpp

```diff
@@ -57,6 +57,7 @@ TEST_F(LoggerTest, Filtering)
     checkEqual("Trace:TRC Trace line logged for 'Trace' component");
 }
 
+#ifndef COVERAGE_ENABLED
```

Comment:
You could make it a bit more compact:
```cpp
#ifndef COVERAGE_ENABLED
TEST_F(LoggerTest, DISABLED_LOGMacro)
#else
TEST_F(LoggerTest, LOGMacro)
#endif
{
...
```
but current version is also fine

Reply:
Both solutions are fine. Keep in mind that `DISABLED_*` will still make the test compile but not run. Given how small the test is it does not really matter too much which way you decide to do it in the end.